chains/neoxt4/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://xt4scan.ngd.network
+    family: blockscout
+    name: xt4scan
+    url: https://xt4scan.ngd.network
+blocks:
+  confirmations: 1
+  estimateBlockTime: 15
+  reorgPeriod: 12
+chainId: 12227332
+deployer:
+  name: Neo Global Development
+  url: https://x.neo.org/
```

Comment:
```suggestion
  url: https://x.neo.org
```

Reply:
Modified in dd41a59e930d91a34b54bb96f3fb58fcfc76b8c0.